src/CommitmentsV2/SFA.DAS.CommitmentsV2/Queries/GetEmployer/GetEmployerResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.CommitmentsV2.Queries.GetEmployer
+{
+    public class GetEmployerResponse
```

Comment:
Similar to api type name comment above, is this name clear enough? Given that the important value is the AccountLegalEntity Name, and is there along with the Account Name, would GetAccountLegalEntity*** be better?